core/src/test/scala/org/apache/spark/benchmark/Benchmark.scala

```diff
@@ -165,7 +165,8 @@ private[spark] class Benchmark(
     val best = runTimes.min
     val avg = runTimes.sum / runTimes.size
```

Comment:
how about just make `avg` as a `Double`?

Reply:
I think I'm understanding what you said. Let me know if you meant something different than I posted

Reply:
Actually, I hope it's `val avg = runTimes.sum.toDouble / runTimes.size`, then `avg` will be more accurate.

Reply:
I will change it. Is it okay that we are changing the value of average (like you pointed out in the other one)?

Reply:
Done! Let me know if you want me to undo it so the value stays the same